admin/RunReports.pl

```diff
@@ -17,13 +18,13 @@
 my $errors = 0;
 for my $name (MusicBrainz::Server::ReportFactory->all_report_names) {
     unless (any { $name =~ /$_/i } @ARGV) {
-        print localtime() . " : Not running $name\n";
+        log_debug { "Not running $name" };
         next;
     }
 
     my $report = MusicBrainz::Server::ReportFactory->create_report($name, $c);
 
-    print localtime() . " : Running $name\n";
+    log_info { "Running $name" };
```

Comment:
Prefixing all messages with `[info]` doesn't look nice or seem very useful.  Debug, warning, and error are the exceptions.  Should we start omitting the log level for informational messages?

Reply:
It doesn’t seem very useful indeed but not that bad either, it can even be useful to grep against: `grep -vF '[info]'`. There is no option in [`Log::Dispatch`](https://metacpan.org/pod/Log::Dispatch) to omit the log level for informational messages, even through callbacks. It would require to extend this class instead of just instantiating it. It looks like an increased complexity for a small eye-candy improvement to me.

Reply:
I thought this is what `_prefix_message` in `MusicBrainz::Server::Log` does -- maybe I'm misreading that code.

I'm sure I could live with it, but the status quo is that we omit `[info]` in most cases (and will continue to omit it for bash scripts).  So if there's an easy way to omit it `_prefix_message` and keep the status quo, that would be nice!